src/test/java/ch/astorm/jotlmsg/io/FlatEntryListStructureTest.java

```diff
@@ -0,0 +1,111 @@
+package ch.astorm.jotlmsg.io;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Iterator;
+import java.util.List;
+
+import org.apache.poi.hsmf.MAPIMessage;
+import org.apache.poi.hsmf.datatypes.ByteChunk;
+import org.apache.poi.hsmf.datatypes.Chunk;
+import org.apache.poi.hsmf.datatypes.MAPIProperty;
+import org.junit.jupiter.api.Test;
+
+public class FlatEntryListStructureTest {
+
+    @Test
+    public void testFlatEntryListStructure1() throws IOException {
+        InputStream msg = OneOffEntryIDStructureTest.class.getResourceAsStream("../msoutlook/replyto.msg");
```

Comment:
Use try-with-resources.

Reply:
Introduced in Java 7 and I have never used it before... Nice.